app/src/renderer/components/governance/ModalDeposit.vue

```diff
@@ -0,0 +1,141 @@
+<template lang="pug">
+  .modal-deposit#modal-deposit(v-click-outside="close")
+    .modal-deposit-header
+      img.icon(class='modal-deposit-atom' src="~assets/images/cosmos-logo.png")
+      span.tm-modal-title Deposit
```

Comment:
not sure if span is the right choice here - but it's ok!

Reply:
what would you have used?